src/bot.ts

```diff
@@ -63,23 +62,24 @@ const run = async () => {
 
   deleted = false
 
-  // INFO: if you don't need lastfm info, comment this bit of code
-  /// lastfm start
-  const currentScrobblingTrackData = await lastfm.call<RecentTracks>('user.getRecentTracks', {
-    user: process.env.LASTFM_USERNAME,
-    limit: 1
-  })
+  let scrobbles: number | undefined = 0
```

Comment:
`number | undefined`, but the initial value is `0`? is `undefined` scenario even possible?